ozkour-back/src/infrastructure/googleslide/googleSlideRepository.js

```diff
@@ -506,6 +506,23 @@ async function copySlide (idPage) {
   }
 }
 
+async function deleteLastSlide () {
+  const res = await getSlides()
+  const idPage = res[1].objectId
```

Comment:
A quoi correspond `res[1]`? Pourquoi **1**?

Reply:
res[1] correspond à la 2eme slide (Commence à l'index 0). On choisit la 2 eme slide car le code duplique la première slide (le template) et la place juste après celle ci (à l'index 1). Or dans le test on veut supprimer la slide que l'on vient de créer pour ne pas laisser de trace.

Reply:
OK ca marche. Dans ce cas. il serait plus clair de creer par exemple une variable:
const indexAddedSlide = 1;
```suggestion
  const idPage = res[1].objectId
```